src/transformers/hf_argparser.py

```diff
@@ -21,13 +21,27 @@
 from enum import Enum
 from inspect import isclass
 from pathlib import Path
-from typing import Any, Callable, Dict, Iterable, List, Literal, NewType, Optional, Tuple, Union, get_type_hints
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Literal,
+    NewType,
+    Optional,
+    Tuple,
+    Union,
+    get_type_hints,
+    TypeVar,
+)
 
 import yaml
 
 
 DataClass = NewType("DataClass", Any)
```

Comment:
Any idea why these `NewType()`s were used originally instead of a simple alias? Like `DataClass = Any`, `DataclassType = Any`.